ui/modal/info_modal_layouts.go

```diff
@@ -59,14 +59,14 @@ func privacyInfo(l *load.Load) []layout.Widget {
 					return ic.Layout(gtx, values.MarginPadding8)
 				}),
 				layout.Rigid(func(gtx C) D {
-					text := l.Theme.Body1("When you turn on the mixer, your unmixed DCRs in this wallet (unmixed balance) will be gradually mixed.")
+					text := l.Theme.Body1("When the mixer is activated, Decred will be gradually transfered from the unmixed account to the mixed account.")
```

Comment:
@raedah mentioned we use `coins` instead of `dcrs`

```suggestion
					text := l.Theme.Body1("When the mixer is activated, coins will be gradually transferred from the unmixed account to the mixed account.")
```

Reply:
I think `funds` is cleaner than `coins`, my initial edit would match closer to what Decrediton has.

Reply:
@raedah stated funds would be good.